src/esp/gfx/TextureVisualizerShader.h

```diff
@@ -46,12 +46,28 @@ class TextureVisualizerShader : public Magnum::GL::AbstractShaderProgram {
   /** @brief Constructor */
   explicit TextureVisualizerShader(Flags flags);
 
+  /**
+   * @brief Build the colorMapTexture_ based on the passed @p colorMap of
+   * colors.
+   *
+   * @param colorMap The map of colors to use
+   * @param wrapEdges Whether queries exceeding the length of @p colorMap will
+   * use the final color, or will wrap around on the texture.
+   * @return Reference to self (for method chaining)
+   */
+  TextureVisualizerShader& setColorMapTexture(
+      Corrade::Containers::ArrayView<const Magnum::Vector3ub> colorMap,
+      bool wrapEdges,
+      float offset,
+      float scale);
+
   /**
    * @brief
    * Offset and scale applied to each input pixel value from depth texture or
    * the ObjectId texture, resulting value is then used to fetch a color from a
-   * color map bound with bindColorMapTexture(). Here we use the Magnum built-in
-   * color map: Magnum::DebugTools::ColorMap::turbo();
+   * color map bound with bindColorMapTexture(). Here we use either the Magnum
```

Comment:
This whole comment is wrong :) Here's no `bindColorMapTexture()`, only `rebindColorMapTexture()`, and the shader is also not responsible for picking the Turbo color map anymore, that's supplied via `setColorMapTexture()`.